tests/test_data/test_pipelines/test_loading/base.py

```diff
@@ -0,0 +1,110 @@
+import os.path as osp
+
+import mmcv
+import numpy as np
+
+
+class ExampleSSNInstance:
```

Comment:
This class is model specified, should not in base. Is it possible to directly import from `ssn_dataset.py`?